packages/lib-classifier/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@zooniverse/classifier",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "_check": "check-dependencies",
+    "build": "npm run _check && webpack --config webpack.dist.js",
+    "dev": "npm run _check && webpack-dev-server --config webpack.dev.js",
+    "start": "npm run _check && PANOPTES_ENV=production npm run dev",
+    "test": "npm run _check && NODE_ENV=test mocha || true"
+  },
+  "author": "Roger Hutchings <roger@hutchin.gs> (https://www.rogerhutchin.gs/)",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@zooniverse/grommet-theme": "^2.0.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "^8.2.3",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^4.1.2",
+    "check-dependencies": "^1.1.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "grommet": "^2.0.0-beta.3",
+    "html-webpack-plugin": "^3.2.0",
+    "jsdom": "^11.11.0",
+    "mobx": "^4.2.1",
+    "mobx-devtools-mst": "^0.9.14",
+    "mobx-react": "^5.1.2",
+    "mobx-state-tree": "^2.0.5",
+    "mocha": "^5.2.0",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2",
+    "sinon": "^5.0.10",
+    "styled-components": "^3.2.6",
+    "superagent": "^3.8.3",
+    "webpack": "^4.8.3",
+    "webpack-cli": "^2.1.3",
+    "webpack-dev-server": "^3.1.4"
+  },
+  "peerDependencies": {
+    "@zooniverse/grommet-theme": "^2.0.0",
+    "@zooniverse/panoptes-js": "^0.1.0",
+    "grommet": "^2.0.0-beta.3",
+    "react": "^16.3.2",
+    "styled-components": "^3.2.6"
+  },
+  "standard": {
+    "globals": [
+      "before",
+      "describe",
+      "expect",
+      "it"
+    ]
+  },
+  "dependencies": {
+    "@zooniverse/panoptes-js": "^0.1.0"
```

Comment:
Should this be listed in the dependencies and the peer deps?

Reply:
Nope, it should be `devDependencies` and `peerDependencies` - good catch